packages/aws-cdk-lib/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -348,6 +366,9 @@ export abstract class QueueProcessingServiceBase extends Construct {
     const defaultScalingSteps = [{ upper: 0, change: -1 }, { lower: 100, change: +1 }, { lower: 500, change: +5 }];
     this.scalingSteps = props.scalingSteps ?? defaultScalingSteps;
 
+    // Setup cooldown
+    this.cooldown = props.cooldown ?? undefined;
```

Comment:
```suggestion
    this.cooldown = props.cooldown;
```

Will naturally turn `this.cooldown` into undefined

Reply:
This doesn't seem like it was resolved